app/views/layouts/shared/_header.html.erb

```diff
@@ -0,0 +1,13 @@
+<header class="navbar navbar-fixed-top navbar-inverse">
+  <div class="container">
+    <%= link_to "sample app", "#", id: "logo" %>
+    <nav>
+      <ul class="nav navbar-nav navbar-right">
+        <li><%= link_to "Home", "/static_pages/home" %></li>
```

Comment:
1. Dùng i18n cho toàn bộ text ở html file nhé. self check các file html khác
2. em tìm hiểu lệnh `rails routes` rồi lấy các prefix path đi, thay vì `"/static_pages/home"` 
ví dụ `home_path` mới chuẩn, ko ai set cứng thế này mô